netlify.toml

```
@@ -81,6 +81,11 @@
   to = "/docs/details/allowlisting-roadie-traffic/"
   force = true
 
+[[redirects]]
+  from = "/docs/getting-started/openapi-specs"
+  to = "/docs/api"
```

We should be redirecting to the trailing slash version of this. Otherwise it will just redirect again. eg https://roadie.io/docs/api

```suggestion
  to = "/docs/api/"
```